proton-c/CMakeLists.txt

```diff
@@ -729,6 +729,12 @@ if (BUILD_PYTHON)
 
 endif (BUILD_PYTHON)
 
+# fuzz tests: tests/fuzz
+option(FUZZ_TEST "Enable building fuzzers and regression testing with libFuzzer" OFF)
+if (FUZZ_TEST)
+  add_subdirectory(src/tests/fuzz)
+endif (FUZZ_TEST)
```

Comment:
This should be probably changed. The fuzz tests should compile always, and switch like this will decide whether to link with actual fuzzer or something like `StandaloneFuzzTargetMain.c` (that would have to be committed here).